### ISSUE TITLE
Deprecated package warnings when installing

### ISSUE THREAD

**Issue by dandv** [NekitCorp/chrome-extension-svelte-typescript-boilerplate#33]
I've just forked and run `npm i` with node v22.11.0, and saw these deprecation messages:

```
$ npm i
npm warn deprecated sourcemap-codec@1.4.8: Please use @jridgewell/sourcemap-codec instead
npm warn deprecated q@1.5.1: You or someone you depend on is using Q, the JavaScript Promise library that gave JavaScript developers strong feelings about promises. They can almost certainly migrate to the native JavaScript promise now. Thank you literally everyone for joining me in this bet against the odds. Be excellent to each other.
npm warn deprecated
npm warn deprecated (For a CapTP with native promises, see @endo/eventual-send and @endo/captp)
```

Anything to worry about?

**Comment by NekitCorp** [comment]
Hi! Sorry for the long reply, I was busy before the new year 😅

This warning comes from the `@crxjs/vite-plugin` package and it was fixed in version [2.0.0-beta.28](https://github.com/crxjs/chrome-extension-tools/releases/tag/%40crxjs%2Fvite-plugin%402.0.0-beta.28). 

I updated all packages in the repository to the latest version and there are no more warnings from npm.
